### ISSUE TITLE
Feature request:Sync state in iCloud

### ISSUE THREAD

**Issue by chrisalibby** [Dimillian/IceCubesApp#2184]
The ability to sync state across devices (where you’ve last read to) would be great.  I assuming this requires saving state data to iCloud.

**Comment by AgileBuckle** [comment]
I agree. It would also be nice to save the customization one has done in the app across iCloud or some other service.

**Comment by 5teveroy** [comment]
Mastodon has a built-in API to sync timelines:
https://docs.joinmastodon.org/methods/markers/
